.github/workflows/ci.yml

```diff
@@ -0,0 +1,91 @@
+---
+name: CI
+'on':
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+env:
+  MAKEFLAGS: -j2
+  ERLC_USE_SERVER: true
+jobs:
+  ci:
+    name: CI OTP ${{matrix.otp_vsn}}, on ${{matrix.os}}
+    runs-on: ${{matrix.os}}
+    strategy:
+      fail-fast: false
+      matrix:
+        otp_vsn: [23, 24, 25, master, latest]
+        os: [ubuntu-18.04, ubuntu-20.04]
+    steps:
+      - name: Update env.
+        run: |
+          sudo sed -i 's/azure\.//' /etc/apt/sources.list # Reduces chance of time-outs
+          sudo apt-get update -y
+          sudo apt-get upgrade -y
+          sudo apt-get install -y --no-install-recommends \
```

Comment:
Removed duplicate `autoconf`...

Reply:
In the bad old days a double autoconf was needed because otherwise certain pre built libraries didn't get built or installed into the right places.

Reply:
Anything required here? Or can I Resolve this conversation?